src/elements/sponsored-by-tag/src/stories.tsx

```diff
@@ -11,12 +11,38 @@ export default {
   title: 'Elements/Sponsored by tag'
 }
 
-export const ExampleWithDefaultText = () => {
-  const providerName: string = text('Provider name', 'Three')
-  const providerLogo: string = text(
-    'Provider logo url',
-    'https://uswitch-mobiles-contentful.imgix.net/kf81nsuntxeb/5eyE4LyswwqIYk0mIsE820/dc0774e3e62d7b39ddeb1729d823a8da/Logo_-_three.png'
+const defaultProviderName = 'Three'
+const defaultProviderLogo =
+  'https://uswitch-mobiles-contentful.imgix.net/kf81nsuntxeb/5eyE4LyswwqIYk0mIsE820/dc0774e3e62d7b39ddeb1729d823a8da/Logo_-_three.png'
+const wideProviderLogo =
+  'https://user-images.githubusercontent.com/15983736/127509419-d54cbc11-3d5b-42ed-9902-6e5a178cd2bc.png'
```

Comment:
This sloppy image reference will be addressed separately here: https://rvu.atlassian.net/browse/MONEY-137